packages/neuron-wallet/package.json

```diff
@@ -22,7 +22,7 @@
     "package-mwl": "electron-builder -mwl",
     "package-win": "electron-builder --win --x64",
     "package-mac": "electron-builder --mac",
-    "package-linux": "electron-builder --linux",
+    "package-linux": "./script/installrpm.sh && electron-builder --linux",
```

Comment:
This script seems only works for macOS, what will happen if I run this on windows and linux ?

Reply:
How about this?

* Add the full `package` script to `/script/package`.
* Only support macOS and Linux (let's forget running this on Windows).
* From that script, check if `rpm` is installed, if not, prompt the user to install.
* Allow passing `PLATFORM=linux` (just example) to that script, so we can leave only one `package` script in `package.json`.